tests/test_routing.py

```diff
@@ -871,7 +871,7 @@ async def my_staticmethod(request: Request) -> None: ...  # pragma: no cover
     def __call__(self, request: Request) -> None: ...  # pragma: no cover
 
 
-@pytest.mark.parametrize(
+@pytest.mark.parametrize(  # pragma: no branch
```

Comment:
I don't know this one as well... 👀

Reply:
Can you try to find out why this works?

Reply:
We can merge the others here first, if you want.

Reply:
Sure, I can try to find out as I will be working on the other cases.

I think we can merge the other ones if it's not a problem.

Reply:
```suggestion
@pytest.mark.parametrize(
```